grails-app/controllers/org/pih/warehouse/api/LocationApiController.groovy

```diff
@@ -110,6 +110,12 @@ class LocationApiController extends BaseDomainApiController {
             } else {
                 throw new ValidationException("Address validation failed", existingLocation.errors)
             }
+            def inventoryLevelInstance = InventoryLevel.findByInventoryAndProductIsNull(existingLocation.inventory)
+            if (!inventoryLevelInstance) {
```

Comment:
this should not be here, because it's not updating location
add a separate endpoint to update the inventory level, because this one has a location object as a parameter, not inventory level